src/App/Web/Polls/Controllers/PollOptionController.php

```
@@ -0,0 +1,48 @@
+use App\Web\Polls\Requests\StorePollOptionRequest;
+use Domain\Events\Models\Event;
+use Domain\Polls\Models\Poll;
+use Domain\Polls\Models\PollOption;
+use Illuminate\Http\RedirectResponse;
+use Illuminate\Support\Str;
+use Illuminate\Validation\ValidationException;
+use Support\Controllers\Controller;
+
+class PollOptionController extends Controller
+{
+    public function store(Event $event, Poll $poll, StorePollOptionRequest $request): RedirectResponse
```

PHPMD (2.15.0)

Avoid unused parameters such as '$event'. (undefined)

(UnusedFormalParameter)